www/Idfa.js

```diff
@@ -2,9 +2,22 @@ var exec = require("cordova/exec");
 var PLUGIN_NAME = "Idfa";
 
 module.exports = {
+    TrackingPermission: Object.freeze({
```

Comment:
In JS word more conventional to have all constant uppercased and without introducing `enum` objects:

```js
module.exports = {
    TRACKING_PERMISSION_AUTHORIZED: 1,
    TRACKING_PERMISSION_DENIED: 2,
    ...
    getInfo: function() {
    ...
}
```